feature-configs/demo/sriov/sriov-daemonset.yaml

```diff
@@ -0,0 +1,66 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: sriov-unsupported-nics-entrypoint
```

Comment:
Sorry, I finally checked.

Under the demo machine we currently have, we are using one of the supported mellanox nics (deviceid of PF 1015).
See also https://docs.openshift.com/container-platform/4.2/networking/multiple-networks/configuring-sr-iov.html#supported-devices_configuring-sr-iov

In theory, we could drop this but in practice it could be useful to run against clusters with non officially supported devices.  I would probably remove this to until we have for example a CI machine that requires it.
What do you think @SchSeba ?

Reply:
works for me